tests/analysis/test_analysis_radialfourier.py

```diff
@@ -65,6 +66,14 @@ def test_sparse(ds_random, lt_ctx):
     lt_ctx.run(analysis)
 
 
+def test_sparse_multi(ds_random, lt_ctx):
```

Comment:
```suggestion
@pytest.mark.with_numba
def test_sparse_multi(ds_random, lt_ctx):
```

Reply:
Thx! I put it on the other test since it is a smaller mask stack, I think.